catalog/camel-report-maven-plugin/pom.xml

```diff
@@ -33,6 +33,10 @@
     <name>Camel :: Catalog :: Camel Report Maven Plugin</name>
     <description>Maven plugin with tools to generate reports on your Camel projects</description>
 
+    <properties>
+        <poi-version>5.2.3</poi-version>
```

Comment:
Can you add poi to parent pom.xml as well

Reply:
Sorry, POI was related to Excel generation. I removed the dependencies and the property.